cluster-up/cluster/kind/common.sh

```diff
@@ -205,6 +213,7 @@ function down() {
     if [ -z $($KIND get clusters | grep ${CLUSTER_NAME}) ]; then
         return
     fi
+    _clean_node_attached_loopdevs
```

Comment:
Unfortunately, we cannot do this as it will wipe out all loopback devices on the system (please correct me if I am wrong) and others may use them (e.g. another instance of kind running on CI).

Can you please confirm if `make cluster-clean` gracefully runs the teardown hook? i.e. if the resource is removed when it is done.

Reply:
> Unfortunately, we cannot do this as it will wipe out all loopback devices on the system (please correct me if I am wrong) and others may use them (e.g. another instance of kind running on CI).

> Can you please confirm if `make cluster-clean` gracefully runs the teardown hook? i.e. if the resource is removed when it is done.

Make cluster-clean can do gracefully exit as it use "kubectl delete", it would trigger preStop hook. And I have verified it. I will use it.

Reply:
Great, thank you!
I guess you will add it as a dependency to the `cluster-down` target in the Makefile, right?

Reply:
> Great, thank you!
> I guess you will add it as a dependency to the `cluster-down` target in the Makefile, right?

Yes